src/components/dataset/pending-submissions.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <modal id="dataset-pending-submissions" :state="state" backdrop hideable @hide="$emit('hide')">
+    <template #title>{{ $t('title') }}</template>
+    <template #body>
+      <div class="modal-introduction">
+        <p>
+          {{ $t('explanation.first') }}
+          <i18n-t tag="span" keypath="explanation.second" :plural="pendingSubmissions">
+            <template #records>
+              <strong>{{ $tcn('explanation.records', pendingSubmissions) }}</strong>
+            </template>
+          </i18n-t>
+        </p>
+        <form id="dataset-auto-convert-form">
+          <div class="radio">
+            <label>
+              <input v-model="convert" name="convert" type="radio" :value="false"
+                aria-describedby="dataset-auto-convert-false">
+              <strong>{{ $t('dontConvert.label') }}</strong>
+            </label>
+            <p id="dataset-auto-convert-false" class="help-block">
+              {{ $tcn('dontConvert.description', pendingSubmissions) }}
+            </p>
+          </div>
+          <div class="radio">
+            <label>
+              <input v-model="convert" name="convert" type="radio" :value="true"
+                aria-describedby="dataset-auto-convert-true">
+              <strong>{{ $t('convert.label') }}</strong>
+            </label>
+            <p id="dataset-auto-convert-true" class="help-block">
+              {{ $tcn('convert.description', pendingSubmissions) }}
+            </p>
+          </div>
+        </form>
+      </div>
+      <div class="modal-actions">
+        <button type="button" class="btn btn-danger" :aria-disabled="convert === null"
+          @click="$emit('success', convert)">
+          {{ $t('action.changeSetting') }}
+        </button>
+        <button type="button" class="btn btn-link" @click="$emit('hide')">
+          {{ $t('action.cancel') }}
+        </button>
+      </div>
+    </template>
+  </modal>
+</template>
+
+<script>
+export default {
+  name: 'DatasetPendingSubmissions'
+};
+</script>
+
+<script setup>
+import { ref } from 'vue';
+
+import Modal from '../modal.vue';
+
+defineProps({
+  state: {
+    type: Boolean,
+    default: false
+  },
+  pendingSubmissions: {
+    type: Number,
+    required: true
+  }
+});
+
+defineEmits(['hide', 'success']);
+
+const convert = ref(null);
+</script>
+
+<i18n lang="json5">
+{
+  "en": {
+    "title": "Pending Submissions",
+    "action": {
+      "changeSetting": "Change setting"
+    },
+    "explanation": {
+      "first": "You are setting Entity creation to occur when Submissions are first received by Central.",
+      "second": "Once this takes effect, Entities will no longer be generated when Submissions are marked Approved, including {records} we found that have neither been marked Approved or Rejected. | Once this takes effect, Entities will no longer be generated when Submissions are marked Approved, including {records} we found that have neither been marked Approved or Rejected.",
```

Comment:
"has" instead of "have" in the first case. Also "nor" instead of "or" in both cases.